pkg/bucketclass/bucketclass.go

```diff
@@ -39,14 +41,96 @@ func Cmd() *cobra.Command {
 // CmdCreate returns a CLI command
 func CmdCreate() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "create <bucket-class-name>",
+		Use:   "create",
 		Short: "Create bucket class",
+	}
+
+	cmd.AddCommand(
+		CmdCreateNamespaceBucketclass(),
+		CmdCreatePlacementBucketClass(),
+	)
+	return cmd
+}
+
+// CmdCreatePlacementBucketClass returns a CLI command
+func CmdCreatePlacementBucketClass() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "placement-bucketclass <bucket-class-name>",
```

Comment:
For caching bucket classes we need both policies, so how to do it from CLI?

```
kubectl create -f bucketclass.yaml
noobaa backingstore create  --placementPolicy  --namespacePolicy
```

Reply:
right, added support for both policied for cache.